app/components/GokuToolbar.android.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+import React, { Component } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+export default class GokuToolbar extends Component {
```

Comment:
Toolbar using .android.js for Android build. unlike iOS where the options are shown using the ActionSheetIOS component, Android shows them straight in the Toolbar using ToolbarAndroid. It uses Icon.ToolbarAndroid from react-native-vector-icons to utilizes icons imported from MateriaIcons.